src/protocol.h

```diff
@@ -253,6 +290,9 @@ enum ServiceFlags : uint64_t {
     // NODE_WITNESS indicates that a node can be asked for blocks and transactions including
     // witness data.
     NODE_WITNESS = (1 << 3),
+    // NODE_COMPACT_FILTERS means the node will service basic block filter requests.
+    // See BIP157 and BIP158 for details on how this is implemented.
+    NODE_COMPACT_FILTERS = (1 << 6),
```

Comment:
`NODE_COMPACT_FILTERS` does not show up in the `getnetworkinfo` RPC's `"localservices"` key, presumably because there's no entry for it in `GetServicesNames`.